plugins/Login/PasswordResetter.php

```diff
@@ -383,7 +395,11 @@ protected function getUserInformation($loginOrMail)
      */
     protected function checkPasswordHash($passwordHash)
     {
-        UsersManager::checkPasswordHash($passwordHash, Piwik::translate('Login_ExceptionPasswordMD5HashExpected'));
+        $hashInfo = $this->passwordHelper->info($passwordHash);
+
+        if (!isset($hashInfo['algo']) || 0 >= $hashInfo['algo']) {
+            throw new Exception(Piwik::translate('Login_ExceptionPasswordMD5HashExpected'));
```

Comment:
@mneudert thanks for the report Marc, just fixed this issue so now this button `View all processed screenshots` should work again!